templates/MongoDB/1/rancher-compose.yml

```diff
@@ -0,0 +1,25 @@
+.catalog:
+  name: "MongoDB"
+  version: "3.0.1-rancher1"
+  description: "MongoDB Replica Set"
+  uuid: mongodb-1
+  minimum_rancher_version: v0.46.0
+  questions:
+    - variable: replset_name
+      description: "Name of the MongoDB replicaSet"
+      label: "ReplicaSet Name"
+      type: "string"
+      required: true
+      default: "rs0"
+    - variable: mongo_version
+      description: "Version of the MongoDB Image</a>"
```

Comment:
This has a random `</a>` in it, and should probably link to the [list of versions](https://hub.docker.com/_/mongo/).

Does mongo handle replicas with different versions of the software well?  Using a generic default like "3" is a recipe for ending up with varying versions.